CollAction/Frontend/app/project/styles/SimpleProjectCreate.scss

```diff
@@ -183,38 +183,4 @@
     color: $coll-action-green;
   }
 
-}
-
-#mobile-carousel {
-  img {
-    margin-bottom: 50px;
-    width: 100%;
-  }
-
-  .carousel-text {
-    margin-bottom: 50px;
-    mark {
-      color: $coll-action-green;
-      background: none;
-      font-weight: bold;
-    }
-  }
-
-  .drop-down {
-    padding: 15px;
-    background-color: $coll-action-green;
-    margin-bottom: 20px;
-    border-radius: 15px;
-    color: #fff;
-    font-size: 18px;
-    
-    i {
-      float: right;
-    }
-
-    span {
-      margin-right: 10px;
-    }
-
-  }
-}
```

Comment:
I think this whole file can be removed; it was for the previous start account page.

Reply:
I've delete this file and tested it 👍